rpm/SPECS/docker-ce-cli.spec

```diff
@@ -35,7 +35,6 @@ BuildRequires: git
 # conflicting packages
 Conflicts: docker
 Conflicts: docker-io
-Conflicts: docker-engine-cs
 Conflicts: docker-ee
 Conflicts: docker-ee-cli
```

Comment:
Hm.. guessing these could also be removed now; it looks like they're already gone from the deb packages, so perhaps we forgot to remove them here 🤔

Reply:
- Opened https://github.com/docker/docker-ce-packaging/pull/1020 as a follow-up, in case it still needed discussion.